plugins/node/opentelemetry-instrumentation-aws-lambda/test/integrations/lambda-handler.force-flush.test.ts

```diff
@@ -23,8 +23,8 @@ import { AwsLambdaInstrumentation } from '../../src';
 import {
   BatchSpanProcessor,
   InMemorySpanExporter,
-} from '@opentelemetry/tracing';
-import { NodeTracerProvider } from '@opentelemetry/node';
+} from '@opentelemetry/sdk-trace-base';
```

Comment:
This dependency is missing in package.json dependencies

Reply:
This is in `devDependencies`. Basically, if the tests passes, then we are good as the dependencies are good.

Reply:
It was not, but it has been added. It was a transitive dependency of something else which is why the tests were passing but it is good practice to declare all of your dependencies.

Reply:
Apparently, I'm blind. Thanks for sharing your perspective.